gradle/dependencies.gradle

```diff
@@ -1,14 +1,14 @@
 buildscript {
   ext.repoDir = new File("$rootProject.buildDir/prebuilts/studio/layoutlib")
 
-  // maps to this commit: https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/fa3aa65
+  // maps to this commit: https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/f5f9f71
   // which runs on JDK 11
-  ext.layoutLibPrebuiltSha = 'fa3aa65'
+  ext.layoutLibPrebuiltSha = 'f5f9f71'
 
   ext.versions = [
     kotlin: '1.6.10',
     agp: '7.1.2',
-    layoutlib: "2021.2.1-patch1-$layoutLibPrebuiltSha",
+    layoutlib: "2022.1.1-canary-$layoutLibPrebuiltSha",
```

Comment:
As discussed in our call a few weeks back, pointing to Electric Eel canary builds would be undesirable since these aren't considered stable yet (Is that a fair statement?).  Dolphin betas might be fine though.

I understand the desire for API 33 support, but any chance the Google Tools team can consider moving that support up to Dolphin release instead?

Or if the desire here is to simply to push a canary repackaged layoutlib to Maven Central, we can certainly do that!

Reply:
Not privy to the mentioned call, but since layoutlib isn't a public API in general wouldn't it be better to just generally target latest? That's sorta what we do with other tools built on unstable APIs (compiler plugins, AGP-oriented gradle plugins, etc).

Reply:
Not everyone feels comfortable with using beta or canary versions (I *do*, but trying to keep the wider community in mind).

Reply:
I think we should land it as is. I believe the risk is very low in doing so. Looking at the [prebuilts repo](https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+log/refs/heads/mirror-goog-studio-main) there has only been [one more drop of prebuilts](https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/2aee3a801ce0b7597a0b72753059d4a8e122019f) with minor fixes. It is using Android T as a base that has shipped as stable to devices and AOSP.

The ship has sailed for AS D, AS EE is close to beta now.  I think we should pick this up.

Reply:
Maybe this canary was relative low change, but as a library maintainer, I wouldn't want to depend on future Google layoutlib canaries being as low change.

Secondly, I don't want to manage several Paparazzi releases in parallel, for each of the AS release channels.

Thirdly, the application package flag change is replaced by a new method -- i believe, in layoutlib-api? -- which is blocked by an sdk-common bump, which in turn is blocked by the resource repository work.

I'll push a new layoutlib if you would like to maintain a fork for a little longer.